README.md

```diff
@@ -0,0 +1,21 @@
+# foo-config-loader
+
+A config loader for the Foo project.
+
+
+# Project tasks
+
+## In order to setup a local env run...:
+
+    virtualenv create -p python3.6 venv/
+    ./venv/bin/pip install pip-tools
```

Comment:
after you create virtualenv you can activate it with a command:
`source ./venv/bin/<virtualenv_name>/activate`
then you can just type `pip install pip-tools` from within the virtualenvironment without specifying path to pip explicitly.

Reply:
Why is this requirement outside requirements.txt file?